site/content/entry/yuzu-progress-report-sep-2023/index.md

```diff
@@ -0,0 +1,379 @@
+- A bug where certain temporary {{< gh-hovercard "11505" "settings would be reset" >}} on emulation start (for example, “Limit emulation speed”) has been fixed.
+- An issue where {{< gh-hovercard "11506" "game shortcuts would appear cropped" >}} on certain devices was fixed.
+- {{< gh-hovercard "11523" "The amount of thread workers for shader compilation" >}} was reduced to one. This is intended to reduce memory consumption.
+- More {{< gh-hovercard "11542" "screen orientation and aspect ratio" >}} misbehaviours were fixed.
+- An issue where the {{< gh-hovercard "11564" "input overlay" >}} would {{< gh-hovercard "11583" "draw offscreen" >}} on some devices was fixed.
+- An issue where the {{< gh-hovercard "11594" "settings activity" >}} would appear dim when in landscape mode while using 3 button navigation was fixed.
+- The in-game menu could be unintentionally opened during the loading animation. {{< gh-hovercard "11597" "This is now fixed." >}}
+- {{< gh-hovercard "11613" "Several crashes" >}} related to the initial setup, emulation activity, and emulation fragment are now fixed.
+- The emulator will now {{< gh-hovercard "11616" "correctly reload the settings file" >}} after resetting all settings.
+- Lastly, a new settings menu {{< gh-hovercard "11603" "unifies all content installation" >}} into a single place. Users can now access the `Manage yuzu data` menu to import/export firmware, saves, user data, and install game content, console, and Amiibo keys. The {{< gh-hovercard "11543" "new User data option" >}} allows users to backup and restore all app data. Useful for example when migrating to other builds, like from a GitHub release to a Play Store release.
+
+{{< single-title-imgs
+    "For the convenience of the lady and the gentleman"
+    "./data1.png"
+    "./data2.png"
+    >}}
+
+## Audio changes
+
+Maide shines here by giving us a {{< gh-hovercard "11460" "rewrite" >}} of the hardware Opus service, responsible for handling decompression of Opus-formatted audio in games.
+The old implementation lacked several functions documented by the reverse engineering community, which is what this rewrite focused on the most.
+
+Now the hardware Opus implementation matches the current information available.
+Maide also took the opportunity to move the processing of Opus to the emulated ADSP (the emulated implementation of the audio coprocessor of the Switch).
+
+The end result of this effort is allowing games that previously faced crashes when trying to decode Opus audio, like `MLB The Show 22 & 23`, the Touhou series games, `Pokemon Quest`, `Pokemon Let's Go Eevee!/Pikachu!`, and `Sea of Stars` to play the audio correctly without crashing.
+
+Maide also fixed audio command processing, avoiding some command lists from returning early before being properly finished.
+The previous implementation would restart the list instead of continuing where it was interrupted, stalling the whole process until it finished properly.
```

Comment:
```suggestion
After an interruption, the previous implementation would restart processing the list from the beginning instead of continuing from where it left off. This stalled audio rendering until it finished properly.
```